camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/creation/BrooklynComponentTemplateResolver.java

```diff
@@ -257,6 +257,7 @@ public boolean canResolve() {
         new BrooklynEntityDecorationResolver.EnricherSpecResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
         new BrooklynEntityDecorationResolver.InitializerResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
         new BrooklynEntityDecorationResolver.SpecParameterResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
+        new BrooklynEntityDecorationResolver.TagsResolver(yamlLoader).decorate(spec, attrs, encounteredRegisteredTypeIds);
```

Comment:
@tbouron what do you mean by a "first-class citizen" here? Do you mean are we right to be promoting it as a first-class citizen, or should we be promoting it more?

It should be a first-class concept - it's not a part of config or anything like that. It needs to end up being added to the `EntitySpec` via `spec.tagsAdd(...)`.